tests/unit/pages/Player.spec.js

```diff
@@ -1,11 +1,6 @@
-import { flushPromises, mount } from "@vue/test-utils";
+import { mount } from "@vue/test-utils";
 import Player from "@/pages/Player.vue";
 import mockAxios from "jest-mock-axios";
-import {
```

Comment:
you can still test a few basic things here:
- if plio emits `loaded`, is plioLoaded called and variable set correctly
- if plio emits `item-toggle`, does it mark item visibility correctly
- does clicking on back button take back